evalml/model_understanding/graphs.py

```diff
@@ -480,33 +492,46 @@ def partial_dependence(pipeline, X, feature, grid_resolution=100):
     elif isinstance(pipeline, evalml.pipelines.MulticlassClassificationPipeline):
         classes = pipeline.classes_
 
-    data = pd.DataFrame({"feature_values": np.tile(values[0], avg_pred.shape[0]),
-                         "partial_dependence": np.concatenate([pred for pred in avg_pred])})
-    if classes is not None:
-        data['class_label'] = np.repeat(classes, len(values[0]))
+    if isinstance(features, int) or isinstance(features, str):
+        data = pd.DataFrame({"feature_values": np.tile(values[0], avg_pred.shape[0]),
+                             "partial_dependence": np.concatenate([pred for pred in avg_pred])})
+        if classes is not None:
+            data['class_label'] = np.repeat(classes, len(values[0]))
+    elif isinstance(features, (list, tuple)):
+        if len(features) == 2:
+            data = pd.DataFrame(avg_pred[0])
```

Comment:
This would only return the partial dependence for the first class for multiclass problems. I think we should either modify this to return the grid for all of the classes or raise an exception that two-way plots are not allowed for multiclass problems.

I don't think modifying this to return the values for all classes would be a heavy lift. I think doing something like this:

```python
df = pd.DataFrame(avg_pred.reshape((-1, avg_pred.shape[-1])))
df.columns = values[1]
df.index = np.tile(values[0], avf_pred.shape[0])
df['classes'] = np.repeat(classes, len(values[0]))
```

should get us most of the way there (though I didn't test 😬 ). The only thing left to modify is how the data is passed to the contour plot.

Reply:
I'm legitimately mystified by bumpy's reshape function and even more by people that can so intuitively use it.  Ok, cool, I'll implement this and try and see if I can follow the one-way multi-class example.